Mathlib/Order/Interval/Finset/Basic.lean

```diff
@@ -308,6 +308,19 @@ section LocallyFiniteOrderTop
 
 variable [LocallyFiniteOrderTop α]
 
+@[simp]
+theorem Ioi_eq_empty_iff : Ioi a = ∅ ↔ IsMax a := by
```

Comment:
The `_iff` is redundant. Same below
```suggestion
theorem Ioi_eq_empty : Ioi a = ∅ ↔ IsMax a := by
```

Reply:
I wanted uniform naming compared to the `Set` version (in `Order/Interval/Set/Basic`)